shared/src/main/scala/drt/shared/airportconfig/Lhr.scala

```diff
@@ -96,7 +96,8 @@ object Lhr extends AirportConfigLike {
     ),
     hasActualDeskStats = true,
     portStateSnapshotInterval = 250,
-    exportQueueOrder = Queues.exportQueueOrderWithFastTrack,
+    forecastExportQueueOrder = Queues.forecastExportQueueOrderWithFastTrack,
```

Comment:
I'm guessing there's a good reason why the forecast queue order includes FT and the desks one doesn't but I can't think of it off the top of my head

Reply:
this was a mistake, thanks for spotting it!